script/deploy/deployAndStoreCREATE3Factory.sh

```
@@ -37,35 +37,36 @@
   echo ""
 
   # deploy CREATE3Factory
+  echo "Trying to deploy CREATE3Factory now"
+  echo ""
   local PRIVATE_KEY=$(getPrivateKey "$NETWORK" "$ENVIRONMENT")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)